app/services/streaming/streaming.ts

```diff
@@ -231,6 +233,19 @@ export class StreamingService extends StatefulService<IStreamingServiceState>
     return moment(this.state.streamingStatusTime);
   }
 
+  private sendReconnectingNotification() {
+    const msg = $t('Stream has disconnected, attempting to reconnect.');
+    const existingReconnectNotif = this.notificationsService.getUnread()
+      .filter((notice: INotification) => notice.message === msg);
+    if (existingReconnectNotif.length !== 0) return;
+    this.notificationsService.push({
+      type: ENotificationType.WARNING,
+      lifeTime: -1,
+      showTime: true,
+      message: $t('Stream has disconnected, attempting to reconnect.')
+    });
```

Comment:
Do you think it's worth automatically clearing this notification if the stream successfully reconnects?

Reply:
wouldn't be against that idea at all